tdesign/src/Upload/Upload.tsx

```diff
@@ -1,6 +1,6 @@
 import { useMemo, CSSProperties } from "react";
-import { Dropzone } from "@splitgraph/react-dropzone-uploader-wrapper";
-import type {
+import {
+  Dropzone,
```

Comment:
Hmm... Should we be importing `Dropzone` from `@mui/material` now, and not from `@splitgraph/react-dropzone-uploader-wrapper`? :thinking: 

Looks like the `Dropzone` MUI component is only being planned ([roadmap](https://mui.com/discover-more/roadmap/#new-components))

Reply:
We're not importing it from MUI – look closer at the diff. The code is this:

```tsx
import {
  Dropzone,
  IFileWithMeta,
  IUploadParams,
  ILayoutProps,
  IInputProps,
  ISubmitButtonProps,
  IDropzoneProps,
} from "@splitgraph/react-dropzone-uploader-wrapper";
```

Reply:
True, my bad. GitHub collapsed a few lines and made it seem like it's a single import

![image](https://user-images.githubusercontent.com/889383/148979479-fc168dca-eefb-46b5-9421-4c392f3ee306.png)